docs/python_docs/python/tutorials/getting-started/crash-course/1-ndarray.md

```diff
@@ -15,113 +15,108 @@
 <!--- specific language governing permissions and limitations -->
 <!--- under the License. -->
 
-# Manipulate data with `ndarray`
+# Step 1: Manipulate data with NP on MXNet
 
-We'll start by introducing the `NDArray`, MXNet’s primary tool for storing and transforming data. If you’ve worked with `NumPy` before, you’ll notice that an NDArray is, by design, similar to NumPy’s multi-dimensional array.
+This getting started exercise introduces the `np` package, which is the primary tool for storing and
+transforming data on MXNet. If you’ve worked with NumPy before, you’ll notice `np` is, by design, similar to NumPy.
 
-## Get started
+## Import packages and create an array
 
-To get started, let's import the `ndarray` package (`nd` is a shorter alias) from MXNet.
 
-```{.python .input  n=1}
-# If you haven't installed MXNet yet, you can uncomment the following line to
-# install the latest stable release
-# !pip install -U mxnet
+To get started, run the following commands to import the `np` package together with the NumPy extensions package `npx`. Together, `np` with `npx` make up the NP on MXNet front end.
 
-from mxnet import nd
+```{.python .input  n=1}
+from mxnet import np, npx
+npx.set_np()  # Activate NumPy-like mode.
```

Comment:
Linking https://github.com/apache/incubator-mxnet/pull/18631 as the line needs removal after https://github.com/apache/incubator-mxnet/pull/18631 is merged